sdk/src/test/java/io/opentelemetry/sdk/trace/export/BatchSpansProcessorTest.java

```diff
@@ -73,6 +75,116 @@ private ReadableSpan createSampledEndedSpan(String spanName) {
     return (ReadableSpan) span;
   }
 
+  @Test
+  public void testConfiguration() {
```

Comment:
Is there any way we can get actually pulling the config from env vars/system properties? I know that's generally a pain, but I don't know if someone had some prior art on making that kind of test to verify that this works for real.

Reply:
Unfortunately, I don't have prior art. But I will search for a proper solution!

Reply:
I've used this before: https://stefanbirkner.github.io/system-rules

Reply:
Thank you @trask 👍